app/src/main/res/values/dimens.xml

```diff
@@ -309,4 +309,22 @@
     <dimen name="robotic_arm_margin_7">7dp</dimen>
     <dimen name="robotic_arm_margin_10">10dp</dimen>
     <dimen name="robotic_arm_height">25dp</dimen>
+    <dimen name="servo_image_width">20dp</dimen>
```

Comment:
Place these close to their respective counterparts. May be you can give a clean up to the whole strings file keeping variables related to a single component close